crates/re_types/src/testing/datatypes/enum_test.rs

```diff
@@ -0,0 +1,203 @@
+// DO NOT EDIT! This file was auto-generated by crates/re_types_builder/src/codegen/rust/api.rs
+// Based on "crates/re_types/definitions/rerun/testing/datatypes/enum.fbs".
+
+#![allow(trivial_numeric_casts)]
+#![allow(unused_imports)]
+#![allow(unused_parens)]
+#![allow(clippy::clone_on_copy)]
+#![allow(clippy::iter_on_single_items)]
+#![allow(clippy::map_flatten)]
+#![allow(clippy::match_wildcard_for_single_variants)]
+#![allow(clippy::needless_question_mark)]
+#![allow(clippy::new_without_default)]
+#![allow(clippy::redundant_closure)]
+#![allow(clippy::too_many_arguments)]
+#![allow(clippy::too_many_lines)]
+#![allow(clippy::unnecessary_cast)]
+
+use ::re_types_core::external::arrow2;
+use ::re_types_core::ComponentName;
+use ::re_types_core::SerializationResult;
+use ::re_types_core::{ComponentBatch, MaybeOwnedComponentBatch};
+use ::re_types_core::{DeserializationError, DeserializationResult};
+
+/// **Datatype**: A test of the enum type.
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub enum EnumTest {
+    /// Great film.
+    Up = 1,
+
+    /// Feeling blue.
+    Down = 2,
+
+    /// Correct.
+    Right = 3,
+
+    /// It's what's remaining.
+    Left = 4,
+
+    /// It's the only way to go.
+    Forward = 5,
+
+    /// Baby's got it.
+    Back = 6,
+}
+
+impl ::re_types_core::SizeBytes for EnumTest {
+    #[inline]
+    fn heap_size_bytes(&self) -> u64 {
+        0
+    }
+
+    #[inline]
+    fn is_pod() -> bool {
+        true
+    }
+}
+
+::re_types_core::macros::impl_into_cow!(EnumTest);
+
+impl ::re_types_core::Loggable for EnumTest {
+    type Name = ::re_types_core::DatatypeName;
+
+    #[inline]
+    fn name() -> Self::Name {
+        "rerun.testing.datatypes.EnumTest".into()
+    }
+
+    #[allow(clippy::wildcard_imports)]
+    #[inline]
+    fn arrow_datatype() -> arrow2::datatypes::DataType {
+        use arrow2::datatypes::*;
+        DataType::Union(
+            std::sync::Arc::new(vec![
+                Field {
+                    name: "_null_markers".to_owned(),
+                    data_type: DataType::Null,
+                    is_nullable: true,
+                    metadata: [].into(),
+                },
```

Comment:
While we're here, add/generate a helper for this somewhere and reduce the code bloat here and for all future enums?
```suggestion
                Field::null("_null_markers")
```

Reply:
I'll test it out first and do it next PR